src/command.c

```diff
@@ -164,7 +165,7 @@ bool cmd_version(target *t, int argc, const char **argv)
 	gdb_outf(", Hardware Version %d\n", platform_hwversion());
 	gdb_out("Copyright (C) 2022 Black Magic Debug Project\n");
 	gdb_out("License GPLv3+: GNU GPL version 3 or later "
```

Comment:
It is, however because of how a "linked work" works under GPL, it is also not inaccurate because on linking, the firmware becomes licensed under the terms of the GPLv3+ which is why this notice is unchanged